configure.ac

```diff
@@ -715,7 +715,7 @@ AS_CASE([$host],
   [*-*-mingw32],
     [AS_CASE([$ocaml_cv_cc_vendor],
       [gcc-[[01234]]-*],
-        [AC_MSG_ERROR(m4_normalize([This version of Mingw GCC is too old.
+        [AC_MSG_ERROR(m4_normalize([This version of MinGW GCC is too old.
```

Comment:
This, the one on L728 and the reference in sys.mli can also be `MinGW-w64`, if you wish

Reply:
@dra27 All done!